src/stores/SettingStore.js

```diff
@@ -9,7 +9,9 @@ let _searchMode = false;
 
 let SettingStore = {
     ...EventEmitter.prototype,
-
+    getTheme() {
```

Comment:
Retrieve theme from UserPreferencesStore and not Settings store, sorry I missed this while making changes :+1: